Britannia/events/BTWK_storyline_events.txt

```diff
@@ -0,0 +1,390 @@
+	theme = skull
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = worry
+	}	
+	lower_left_portrait = {
+		character = character:5  # Mordred ap Mordred
+		animation = no
+	}
+	lower_right_portrait = {
+		character = character:2  # Merlin
+		animation = no
+	}
+	lower_center_portrait = {
+		character = character:20  # Tewdric
+		animation = no
+	}
+
+	
+	option = {
+		name = BTWK_story_event.0001.a
+		trigger = {
+			NOT = { has_character_flag = utherillness }
+			is_alive = yes	
+		}
+		character:1 = {
+			add_trait = pneumonic
+			add_character_flag = utherillness
+			trigger_event = {
+				id = BTWK_story_event.0002
+				days = {100 180}
+			}
+		}
+		
+	}
+}
+
+BTWK_story_event.0002 = {  # Should probably change this to make it non-hidden & a unique end of life event for Uther.
+	type = character_event
+	title = BTWK_story_event.0002.t
+	desc = BTWK_story_event.0002.desc
+	# More fitting music cue/sounds needed
+	theme = skull
+
+
+	right_portrait = {
+		character = character:1
+		animation = worry
+	}
+	
+	option = {
+		name = BTWK_story_event.0002.a
+		trigger = {
+			this = character:1	
+			has_character_flag = utherillness
+			is_alive = yes
+			NOT = { has_character_flag = utherincapable }
+		}
+		character:1 = {
+			add_trait = incapable
+			add_character_flag = utherincapable
+		}
+		
+	}
+}
+
+BTWK_story_event.0003 = {  #Sub-event that delays Arthur's return, & fires "Uther is dead" event.
+
+	type = character_event
+	hidden = yes
+	
+	trigger = {
+		character:1 = { is_alive = no }
+		has_global_variable = uther_is_dead
+	}
+	immediate = {
+		remove_global_variable = uther_is_dead
+		#  Fires Arthur's return HIDDEN EVENT  #
+		trigger_event = {
+			id = BTWK_story_event_0004
+		}
+	}
+}
+
+BTWK_story_event.0004 = {
+
+	type = character_event
+	hidden = yes
+
+	immediate = {
+
+		#  Arthur gets d_durnac & vassals  #
+		create_title_and_vassal_change = {
+			type = granted
+			save_scope_as = arthur_pendragon_durnac
+			add_claim_on_loss = yes
+		}
+		title:d_centdumnonia = {
+			every_de_jure_county = {
+				limit = {
+					holder = character:5
+				}
+				change_title_holder = {
+					holder = character:3
+					change = scope:arthur_pendragon_durnac
+				}
+			}
+			change_title_holder = {
+				holder = character:3
+				change = scope:arthur_pendragon_durnac
+			}
+			every_de_jure_county_holder = {
+				change_liege = {
+					liege = character:3
+					change = scope:arthur_pendragon_durnac
+				}
+			}
+		}
+		resolve_title_and_vassal_change = scope:arthur_pendragon_durnac
+		#####################################
+
+		# Will need to add liege change when King Ban added, OR, destroy Arthur's Company, then change employer of all chars in Arthur's Company.
+		character:3 = {
+			add_character_flag = arthur_pendragon
+			give_nickname = nick_pendragon
+		}
+		character:5 = {
+			add_character_flag = mordred_pendragon
+		}
+
+		#  Fires "Uther is dead & Arthur returns" event  #
+		every_ruler = {
+			if = {
```

Comment:
We actually don't need this if block in here - we can use limit blocks inside every_X = { scopes directly